src/main/Core/FileSystemUtility/Contract/FileSystemUtility.ts

```diff
@@ -4,6 +4,9 @@ export interface FileSystemUtility {
     pathExists(fileOrFolderPath: string): Promise<boolean>;
     readJsonFile<T>(filePath: string): Promise<T>;
     readJsonFileSync<T>(filePath: string): T;
+    readIniFile(filePath: string): Promise<Record<string, Record<string, string>>>;
+    readIniFileSync(filePath: string): Record<string, Record<string, string>>;
```

Comment:
This is very specific to ini files, I suggest to create a separate utility for reading ini files.